app/src/main/java/com/codesoom/assignment/controllers/ProductController.java

```diff
@@ -1,55 +1,101 @@
 package com.codesoom.assignment.controllers;
 
+import com.codesoom.assignment.application.AuthenticationService;
 import com.codesoom.assignment.application.ProductService;
 import com.codesoom.assignment.domain.Product;
 import com.codesoom.assignment.dto.ProductData;
+import com.codesoom.assignment.utils.JwtUtil;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
 import java.util.List;
 
+/**
+ * 상품에 대한 요청을 처리합니다.
+ */
 @RestController
 @RequestMapping("/products")
 @CrossOrigin
 public class ProductController {
     private final ProductService productService;
+    private final AuthenticationService authenticationService;
 
-    public ProductController(ProductService productService) {
+    public ProductController(ProductService productService, AuthenticationService authenticationService) {
         this.productService = productService;
+        this.authenticationService = authenticationService;
     }
 
+    /**
+     * 상품 전체 목록에 대한 요청을 처리합니다.
+     *
+     * @return 상품 목록
+     */
     @GetMapping
     public List<Product> list() {
         return productService.getProducts();
     }
 
+    /**
+     * 상품에 대한 정보 요청을 처리합니다.
+     *
+     * @param id 상품 식별자
+     * @return 상품
+     */
     @GetMapping("{id}")
     public Product detail(@PathVariable Long id) {
         return productService.getProduct(id);
     }
 
+    /**
+     * 상품 생성에 대한 요청을 처리합니다.
```

Comment:
이렇게 하면 어떨까요? 좀 더 이해하기 쉬울까요?
```suggestion
     * 새로운 상품을 생성하고, 생성된 상품 정보를 리턴합니다.
```